src/engine/configuration.cpp

```diff
@@ -76,6 +76,19 @@ namespace {
         // OpenSpace. If it is not provided, it defaults to 'User'
         std::optional<Visibility> propertyVisibility;
 
+        enum class [[codegen::map(openspace::OpenSpaceEngine::PropertyConfirmation)]]
+        PropertyConfirmation
+        {
+            Never,
+            Default,
+            Always
+        };
+        // Determines when the property confirmation modal should be shown when starting
+        // up OpenSpace. If it is not provided, it defaults to 'Default', i.e., each
+        // individual property determines whether it requires a confirmation or not
+        std::optional<PropertyConfirmation> propertyConfirmation;
```

Comment:
The first sentence sounds a bit weird here. How about something like: 
```suggestion
        // Determines the setting for when property confirmation modals should be shown
        // on property value changes. If not provided, it defaults to 'Default', i.e.,
        // each individual property determines whether it requires a confirmation or not.
        std::optional<PropertyConfirmation> propertyConfirmation;
```

Not that we have decided that these descriptions should end with punctuation. It's just not applied to the other parameters in this file yet.